src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,137 @@
+define([
+    // Change this to filer vs. filer.min if you need to debug Filer
+    "thirdparty/filer/dist/filer.min",
+],function(Filer){
+    "use strict";
+
+    var ProjectStats = {};
+    
+    var _Path = Filer.Path;
+    var _root;
+
+    // keep track of project state
+    // host cache in the object should any other project states have to be added
+    var _cache = {};
+
+    var _fs = new Filer.FileSystem();
+    var _shell = new _fs.Shell();
+    ProjectStats.getFileSystem = function(){
+        return _fs;
+    };
+    // NOTE: THIS WILL DESTROY DATA! For error cases only, or to wipe the disk.
+    ProjectStats.formatFileSystem = function(flags, callback){
+        _fs = new Filer.FileSystem(flags, callback);
```

Comment:
@humphd I am not really sure if that is required based of the original code that avoids doing it. [Here](https://github.com/mozilla/brackets/blob/master/src/bramble/client/main.js#L118)